pkg/route/engine.go

```diff
@@ -580,7 +584,10 @@ func (engine *Engine) addRoute(method, path string, handlers app.HandlersChain)
 	utils.Assert(method != "", "HTTP method can not be empty")
 	utils.Assert(len(handlers) > 0, "there must be at least one handler")
 
-	debugPrintRoute(method, path, handlers)
+	if debugPrintRouteEnable(engine.options) {
```

Comment:
why use engine.options.***  to judge

Reply:
> why use engine.options.*** to judge instead define function ???
> 
> ```
> if engine.options.DebugPrintRoute{
> 
> }
> ```

At first it was for consistency with previous code, but your judgement method is better. Corrected.